pkg/infra/usagestats/types.go

```diff
@@ -0,0 +1,10 @@
+package usagestats
+
+type ConcurrentUsersStats struct {
```

Comment:
Looks as if we can avoid exporting this type outside of the package:
```suggestion
type concurrentUsersStats struct {
```